api/src/main/resources/liquibase.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+ 
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9
+                  http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+ 
+    <!--
+    	See http://wiki.openmrs.org/display/docs/Module+liquibase+File for 
+    	documentation on this file.
+    
+        See http://www.liquibase.org/manual/home#available_database_refactorings
+        for a list of supported elements and attributes
+    -->
+
+    <changeSet id="create-complete-program-task-2025-11-04" author="siddharth">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="0">
+                SELECT COUNT(*) FROM scheduler_task_config
+                WHERE schedulable_class = 'org.openmrs.module.lamp.scheduler.CompleteProgramsTask'
+                And name = 'Complete LAMP Program Task'
+            </sqlCheck>
+        </preConditions>
+        <comment>Inserting CompletePrograms Task into 'schedule_task_config' table</comment>
+        <insert tableName="scheduler_task_config">
```

Comment:
At some point, I would like us to migrate from using this approach (built-in scheduled task engine). If the module is removed for some reason, there will be always scheduled task configuration hanging around, which throws errors due to classes not being found